frc_msgs/msg/CANStatus.msg

```diff
@@ -0,0 +1,7 @@
+# Corresponds to frc::CANStatus in RobotController.h
+Header header
+float32 percent_bus_utilization
```

Comment:
Float 32 or 64

Reply:
Depends on the field. All fields have now been updated to match the return type from the corresponding wpilib function.